test/cmd/create.sh

```diff
@@ -85,7 +85,12 @@ run_kubectl_create_error_tests() {
   rm "${ERROR_FILE}"
 
   # Posting a pod to namespaces should fail.  Also tests --raw forcing the post location
-  grep -q "cannot be handled as a Namespace: converting (v1.Pod)" <<< "$( kubectl convert -f test/fixtures/doc-yaml/admin/limitrange/valid-pod.yaml -o json | kubectl create "${kube_flags[@]}" --raw /api/v1/namespaces -f - --v=8 2>&1 )"
+  # json
+  kubectl apply "${kube_flags[@]}" -f test/fixtures/doc-yaml/admin/limitrange/valid-pod.yaml
+  grep -q 'cannot be handled as a Namespace: converting (v1.Pod)' <<< "$( kubectl get "${kube_flags[@]}" pod/valid-pod -o json | kubectl create "${kube_flags[@]}" --raw /api/v1/namespaces -f - --v=8 2>&1 )"
+  kubectl delete "${kube_flags[@]}" pod/valid-pod
```

Comment:
You're unnecessarily creating the pod and then removing, just change this so that the get is not needed, b/c it's not. 
Iow:
```bash
kubectl create "${kube_flags[@]}" -f test/fixtures/doc-yaml/admin/limitrange/valid-pod.yaml --raw /api/v1/namespaces --v=8
```
will be sufficient b/c you'll try to create a pod against endpoint for namespaces which will return you just that error.

Reply:
I see. Fixed.